src/App.css

```diff
@@ -1,66 +1,19 @@
-.App {
-  text-align: center;
+.error input {
```

Comment:
All of these error css will eventually come from the dotgov components library but for now this is where they will linger to bypass the citysourced css.